cnf-certification-test/identifiers/identifiers.go

```diff
@@ -761,6 +770,14 @@ the changes for you.`,
 		Remediation:           `ensure the CNF apps don't listen on undeclared containers' ports`,
 		BestPracticeReference: bestPracticeDocV1dot2URL + " Section 16.3.1.1",
 	},
+	TestUndeclaredContainerPortsUsage: {
```

Comment:
Same here. You don't need to add it again, as this catalog entry already exists on line 765. Please remove it.